IDEAS/Buildings/Components/Shading/Interfaces/Records/Box.mo

```diff
@@ -0,0 +1,6 @@
+within IDEAS.Buildings.Components.Shading.Interfaces.Records;
```

Comment:
I'm confused.. I see these models nowhere in IDEAS in dymola?

Reply:
To clarify, I think these records are old files that were not removed properly when developing the model. They are not used so they can be removed.